### ISSUE TITLE
Error doing job: Task exception was never retrieved

### ISSUE THREAD

**Issue by haaino** [golles/ha-knmi#53]
### Did you read the instructions?

- [X] I have read the [README](../blob/main/README.md), including the [known-limitations](../blob/main/README.md#known-limitations) section.

### The problem

I’m getting error messages in my log file concerning images that cannot be retrieved. 

### What version of this integration has the issue?

1.3.2

### What version of Home Assistant Core has the issue?

HA 2022.11.3

### Home Assistant log

```shell
This error originated from a custom integration.

Logger: homeassistant
Source: custom_components/knmi/weather.py:102 
Integration: KNMI (documentation, issues) 
First occurred: 11:13:21 (100 occurrences) 
Last logged: 13:02:51

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 813, in _update_entity_states
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 532, in async_update_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 583, in _async_write_ha_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 551, in _stringify_state
    if (state := self.state) is None:
  File "/usr/src/homeassistant/homeassistant/components/weather/__init__.py", line 832, in state
    return self.condition
  File "/config/custom_components/knmi/weather.py", line 102, in condition
    return CONDITIONS_MAP[self.coordinator.get_value("image")]
KeyError: 'lichtbewolkt'
```

### Additional information

Home Assistant 2022.11.3
Supervisor 2022.10.2
Operating System 9.3
Frontend 20221108.0 - latest

**Comment by golles** [comment]
Will be fixed once #52 is merged and released

**Comment by golles** [comment]
https://github.com/golles/ha-knmi/releases/tag/1.3.3
